### PR TITLE
Create map view & enable location

### PR DESCRIPTION
**Pull request by aelalamy42**
This pull request create the Maps Fragment for the app, using the Maps API.

This map is styled according to the color schema already established for the app.

A location permission is also enabled. If the location is not authorized, the map is centered at EPFL. Otherwise, the location of the user is shown and the maps stays centered around that position.

Screenshot from 2023-03-14 10-43-58

### CODE REVIEW

File: app/src/main/java/ch/epfl/culturequest/MapsActivity.java
Comment (TheGuyWithoutH) [inline app/src/main/java/ch/epfl/culturequest/MapsActivity.java:1]: Why letting this activity if the fragment is there now?

File: app/src/main/java/ch/epfl/culturequest/ui/map/MapsViewModel.java
Comment (TheGuyWithoutH) [inline app/src/main/java/ch/epfl/culturequest/ui/map/MapsViewModel.java:1]: Maybe add documentation for such public methods in the style we learned previously.

File: build.gradle
Comment (TheGuyWithoutH) [inline build.gradle:1]: Are you sure of these changes?

File: app/src/main/java/ch/epfl/culturequest/ui/map/MapsViewModel.java
Comment (aelalamy42) [reply app/src/main/java/ch/epfl/culturequest/ui/map/MapsViewModel.java:1]: Done!

File: build.gradle
Comment (aelalamy42) [reply build.gradle:1]: Somethings were not necessary, I reviewed everything and kept only things that I actually needed

**Review by TheGuyWithoutH — APPROVED** [review]
Good to go!

**Pull request merged**